extensions/shared/blocks/cover/cover-replace-control-button.js

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useBlockEditContext } from '@wordpress/block-editor';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { useRef, useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { CoverMediaContext } from './components';
+import { isUpgradable, isVideoFile } from "./utils";
+
+export default createHigherOrderComponent( MediaReplaceFlow => props => {
+	const { name } = useBlockEditContext();
+	const preUploadFile = useRef();
+	if ( ! isUpgradable( name ) ) {
```

Comment:
Would this be enough?
 
```suggestion
	if ( 'core/cover' !== name ) {
```